generators/lib/prompts.js

```diff
@@ -66,6 +66,22 @@ var prompts = [
     choices: function(answers) {
       return answers['drupalDistro'].versions;
     }
+  },
+  {
```

Comment:
It might be verbose, but maybe we should break this up into asking if we should generate a profile, then demanding a name for the profile. That way we can add more prompts in the future.